packages/blade/src/components/Badge/__tests__/__snapshots__/Badge.native.test.tsx.snap

```diff
@@ -49,75 +49,83 @@ exports[`<Badge /> should render Badge with Icon 1`] = `
         ]
       }
     >
-      <RNSVGSvgView
-        accessibilityElementsHidden={true}
-        align="xMidYMid"
-        bbHeight="12px"
-        bbWidth="12px"
-        fill="none"
-        focusable={false}
-        height="12px"
-        importantForAccessibility="no-hide-descendants"
-        meetOrSlice={0}
-        minX={0}
-        minY={0}
+      <View
```

Comment:
I think we should avoid adding a wrapper to every icon. Can't we inject styling directly into the SVG element?

Reply:
Made the change in other PR